lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -844,7 +847,9 @@ public float getScale() {
 
   @Override public void setScaleType(ScaleType scaleType) {
     super.setScaleType(scaleType);
-    lottieDrawable.setScaleType(scaleType);
+    if (lottieDrawable != null) {
+      lottieDrawable.setScaleType(scaleType);
```

Comment:
@boduan402 Could this cause the scale type to never be set?

Reply:
It won't. 
- For case user config scaleType via XML, the lottieDrawable is null because it is called from super class's constructor. But the scaleType will be set into LottieDrawable at line 208 which is inside init() method.
- For case user set ScaleType through API call, the lottieDrawable is NonNull.